exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity <= 25; // finish this statement
```

Comment:
Can you edit the statement by assigning the `studentCount` and `mentorCount` variables to the `enoughSpaceInRoom` variable and use a comparison operator and the `roomMaxCapacity` to make the statement make sense?